### DISCUSSION TITLE
macOS 12 Monterey

### DISCUSSION THREAD

**Discussion by SkyrilHD** [SkyrilHD/Dell-E7250-Hackintosh#27] [Show and tell]
The EFI Update 0.7.0 was delayed because I wanted to see if macOS 12 Monterey works. And oh boy it does. For a list of working features check my Projects page here: https://github.com/SkyrilHD/Dell-E7250-Hackintosh/projects/7

Looks very smooth :P

If you guys also want to boot to Monterey, the only changes I made was adding -lilubetaall and -vsmcbeta to boot-args.

Hopefully I will update the EFI tomorrow. Only time will tell.

![Screenshot 2021-06-07 at 16 11 38](https://user-images.githubusercontent.com/28839925/121099291-dd627980-c7f7-11eb-8877-5ceaf265e551.png)

**Comment by ghost** [comment]
Wow! Props to you for doing this so quickly! I have a question. Does the latest release boot macOS Monterey?

**Reply by ghost** [reply]
Nvm my question, I found the answer (it boots up) Thanks again

**Comment by n2ng-dev** [comment]
Wow
Waiting for it,

I got random restart when open setting menu,
Use EFI newest and boot arg as you say

**Reply by SkyrilHD** [reply]
"setting menu"? what is that?

**Reply by n2ng-dev** [reply]
When I Click restart always hang and then force restart

**Reply by SkyrilHD** [reply]
during installation?

**Reply by n2ng-dev** [reply]
After Done installation,

**Reply by SkyrilHD** [reply]
hmmm I don't have this issue. Maybe it is macOS' fault, wait for new beta update which might fix the issue.

**Reply by n2ng-dev** [reply]
Can you upload EFI folder for Monterey ?

**Reply by SkyrilHD** [reply]
it is the same as v2.7 just added `-lilubetaall -vsmcbeta` as boot-arg https://github.com/SkyrilHD/Dell-E7250-Hackintosh/blob/14daa31a1ebd2eae43e4b57b47cadb04eace1f7d/OpenCore/EFI/OC/config.plist#L1228-L1229

**Reply by n2ng-dev** [reply]
I use v2.7 and boot arg, but I got random restart too.,

Boot second stage a little longer than usual

**Reply by SkyrilHD** [reply]
try disabling itlwm, IntelBluetoothInjector and IntelBluetoothFirmware

**Reply by n2ng-dev** [reply]
Still got random restart, disabling itlwm, IntelBluetoothInjector and IntelBluetoothFirmware

**Reply by n2ng-dev** [reply]
Please update for Monterey

**Comment by devsurajrai** [comment]
Wow ! Really Excited for it . Thanks,Man !

**Comment by 83435** [comment]
Bro how to create boot loader for Monterey
